specification/search/data-plane/Azure.Search/preview/2020-06-30-Preview/searchservice.json

```diff
@@ -5788,14 +5788,253 @@
+        },
+        "failOnUnsupportedContentType": {
+          "type": "boolean",
+          "default": false,
+          "description": "For Azure blobs, set to false if you want to continue indexing when an unsupported content type is encountered, and you don't know all the content types (file extensions) in advance."
+        },
+        "failOnUnprocessableDocument": {
+          "type": "boolean",
+          "default": false,
+          "description": "For Azure blobs, set to false if you want to continue indexing if a document fails indexing."
+        },
+        "indexStorageMetadataOnlyForOversizedDocuments": {
+          "type": "boolean",
+          "default": false,
+          "description": "For Azure blobs, set this property to true to still index storage metadata for blob content that is too large to process. Oversized blobs are treated as errors by default. For limits on blob size, see https://docs.microsoft.com/azure/search/search-limits-quotas-capacity."
+        },
+        "delimitedTextHeaders": {
+          "type": "string",
+          "description": "For CSV blobs, specifies a comma-delimited list of column headers, useful for mapping source fields to destination fields in an index."
+        },
+        "delimitedTextDelimiter": {
+          "type": "string",
+          "description": "For CSV blobs, specifies the end-of-line single-character delimiter for CSV files where each line starts a new document (for example, \"|\")."
+        },
+        "firstLineContainsHeaders": {
+          "type": "boolean",
+          "default": true,
+          "description": "For CSV blobs, indicates that the first (non-blank) line of each blob contains headers."
+        },
+        "documentRoot": {
+          "type": "string",
+          "description": "For JSON arrays, given a structured or semi-structured document, you can specify a path to the array using this property."
+        },
+        "dataToExtract": {
+          "$ref": "#/definitions/DataToExtract"
+        },
+        "imageAction": {
+          "$ref": "#/definitions/ImageAction"
+        },
+        "allowSkillsetToReadFileData": {
+          "type": "boolean",
+          "default": false,
+          "description": "If true, will create a path //document//file_data that is an object representing the original file data downloaded from your blob data source.  This allows you to pass the original file data to a custom skill for processing within the enrichment pipeline, or to the Document Extraction skill."
+        },
+        "pdfTextRotationAlgorithm": {
+          "$ref": "#/definitions/PdfTextRotationAlgorithm"
+        },
+        "executionEnvironment": {
+          "$ref": "#/definitions/ExecutionEnvironment"
+        },
+        "queryTimeout": {
+          "type": "string",
```

Comment:
This is not a valid ISO8601 duration - I assume that the service has a custom duration format?

Reply:
The service takes the format "hh:mm:ss" (documented at https://docs.microsoft.com/en-us/rest/api/searchservice/create-indexer#other-configuration-parameters).  According to https://en.wikipedia.org/wiki/ISO_8601#Times, it looks like this is valid.  From that page: "ISO 8601-1:2019 allows the T to be omitted in the extended format, as in "13:47:30"."  Is this acceptable, @johanste?  If not, please let me know how you'd like to move forward.